modules/oe_whitelabel_search/oe_whitelabel_search.module

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @file
+ * OE Whitelabel Search Module.
+ */
+
+declare(strict_types =  1);
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter() for facets_forms.
+ */
+function oe_whitelabel_form_facets_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (isset($form['actions'])) {
+    $action_classes = ['mt-4', 'd-grid', 'gap-4', 'd-md-block'];
+    foreach ($action_classes as $action_class) {
+      if (!isset($action_class, $form['actions']['#attributes']['class']) || !in_array($action_class, $form['actions']['#attributes']['class'], TRUE)) {
+        $form['actions']['#attributes']['class'][] = $action_class;
+      }
+    }
+  }
+  if (isset($form['actions']['submit'])) {
+    $submit_classes = ['me-md-3', 'btn', 'btn-primary', 'btn-md'];
+    foreach ($submit_classes as $submit_class) {
+      if (!isset($submit_class, $form['actions']['submit']['#attributes']['class']) || !in_array($submit_class, $form['actions']['submit']['#attributes']['class'], TRUE)) {
+        $form['actions']['submit']['#attributes']['class'][] = $submit_class;
+      }
+    }
+  }
+  if (isset($form['actions']['reset'])) {
+    $reset_classes = ['btn', 'btn-light', 'btn-md'];
+    foreach ($reset_classes as $reset_class) {
+      if (!isset($form['actions']['reset']['#attributes']['class']) || !in_array($reset_class, $form['actions']['reset']['#attributes']['class'], TRUE)) {
+        $form['actions']['reset']['#attributes']['class'][] = $reset_class;
+      }
+    }
+  }
+}
+
+/**
+ * Helper function to preprocess the facets_summary item list.
+ *
+ * @param array $variables
```

Comment:
remove, hooks and preprocesses are like interfaces for drupal, so no need to define the variables as they are already documented